pages/TheLearnerView.html

```diff
@@ -20,7 +20,7 @@
     sequence.
   </p>
 
-  <img src="static/images/exploration.png" width="620">
+  <img src="static/images/exploration2.png" width="620">
```

Comment:
Image link is broken. The other images don't seem to have been updated?

Note: for rating the exploration, please add some text indicating that the learner has to be logged-in before they can rate an exploration.